k-distribution/src/main/scripts/lib/pyk/kast.py

```diff
@@ -95,29 +155,28 @@ def KRequire(krequire):
 def isKRequire(k):
     return k["node"] == "KRequire"
 
-def KDefinition(mainModule, modules, requires = None, atts = None):
-    return { "node": "KDefinition", "mainModule": mainModule, "modules": modules, "requires": requires, "atts": atts }
+def KDefinition(mainModule, modules, requires = None, att = None):
+    return { "node": "KDefinition", "mainModule": mainModule, "modules": modules, "requires": requires, "att": att }
 
 def isKDefinition(k):
     return k["node"] == "KDefinition"
 
 def isCellKLabel(label):
     return len(label) > 1 and label[0] == "<" and label[-1] == ">"
 
-def addAttributes(kast, atts):
+def addAttributes(kast, att):
     if isKRule(kast):
         newAtts = []
-        if kast["atts"] is None:
-            newAtts = atts
+        if kast["att"] is None:
+            newAtts = att
         else:
-            newAtts.extend(kast["atts"])
-        return KRule(kast["rule"], requires = kast["requires"], ensures = kast["ensures"], atts = newAtts)
+            newAtts.extend(kast["att"])
+        return KRule(kast["body"], requires = kast["requires"], ensures = kast["ensures"], att = newAtts)
```

Comment:
Does the `att` parameter get added to `newAtts` when the else branch is taken?

Reply:
Hmmm, looks like some bugs in attributes in general slipped through, I've updated.